virtual_labs/infrastructure/kc/auth.py

```diff
@@ -63,7 +64,17 @@ def verify_jwt(
                 message="Session not active",
                 detail="Session is dead or user not found",
             )
-
+    except KeycloakError as exception:
+        logger.error(
+            f"Keyclock error while token introspection {exception.error_message} BODY: {exception.response_body} CODE: {exception.response_code}"
+        )
```

Comment:
sugg: u can use `exception.__str__` will return the same

Reply:
on it

Reply:
Thanks for the suggestion. It's indeed cleaner. I've made the changes.